From f4590382ab713e95ab3a70980ee4d99165da1335 Mon Sep 17 00:00:00 2001
From: razorloves <razorloves@gmail.com>
Date: Fri, 4 May 2018 23:14:28 +0200
Subject: [PATCH 158/249] Revert "recovery_install_heimdall: flash recovery to
 BOOT and RECOVERY partition"

A couple steps later it says to make a backup in twrp, so this is
not a good situation for new users since that backup won't boot.

This reverts commit b1a7cc6a4d70b8017516d6af8a49be594b5665fe.

Change-Id: I2eb472c4432edb895d73d75145999475d9b8c531
---
 _includes/templates/recovery_install_heimdall.md | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/_includes/templates/recovery_install_heimdall.md b/_includes/templates/recovery_install_heimdall.md
index 3925d75..c2ef8eb 100644
--- a/_includes/templates/recovery_install_heimdall.md
+++ b/_includes/templates/recovery_install_heimdall.md
@@ -57,8 +57,12 @@ The preferred method of installing a custom recovery is through this boot mode{%
     Accept the disclaimer, then insert the USB cable into the device.
 4. On the computer, open a command prompt (on Windows) or terminal (on Linux or macOS) window in the directory the recovery image is located, and type:
 
-        heimdall flash --BOOT twrp-x.x.x-x-{{ twrp_codename }}.img --RECOVERY twrp-x.x.x-x-{{ twrp_codename }}.img
+        heimdall flash --RECOVERY twrp-x.x.x-x-{{ twrp_codename }}.img --no-reboot
 
     {% include tip.html content="The file may not be named identically to what stands in this command, so adjust accordingly. If the file is wrapped in a zip or tar file, extract the file first, because Heimdall is not going to do it for you." %}
 5. A blue transfer bar will appear on the device showing the recovery being transferred.
-6. The device will reboot into recovery.
+6. Unplug the USB cable from your device.
+7. Manually reboot into recovery:
+    * {{ site.data.devices[page.device].recovery_boot }}
+
+    {% include note.html content="Be sure to reboot into recovery immediately after having installed the custom recovery. Otherwise the custom recovery will be overwritten and the device will reboot (appearing as though your custom recovery failed to install)." %}
-- 
2.17.1

